text/rustconf-steering-committee.md

```diff
@@ -5,46 +5,49 @@
 
 # Summary
 
-This RFC proposes creating a RustConf steering committee to represent the Rust Project’s perspectives and priorities throughout the RustConf 2024 planning process. 
+This RFC proposes creating an advisory Rust Steering Committee for RustConf to represent Rust Project and community perspectives and priorities throughout the RustConf 2024 planning process. 
```

Comment:
If this RFC is purposefully limited just to the 2024 RustConf than I'm not sure an RFC is appropriate. 

My initial impression of what this document would be was a document for establishing how RustConf would be run at a high-level. Such a document would clearly layout who has authority over which decisions. For example, I imagined that the RFC would have said something like: 

## Areas of Responsibility

The following groups are responsible for the following areas. More detail on the nature and limits of the responsibility can be found below:
* The Rust Foundation Staff: logistical operation of the event itself, coordinating sponsor relations, ...
* The Steering Committee: all programming aspects of the event (e.g., program committee, MC selection, format of talks, whether to include workshops, inclusion of an UnConf day(s) or other supplementary events, etc.)